### PR TITLE
Add a page with blog posts list

### PR DESCRIPTION
**Pull request by makkaronis4e**
Improve/create files for url page for django blog

#11

### CODE REVIEW

**Comment by makkaronis4e** [comment]
@webknjaz HELLO new PR here!!! pew pew pew BANG BANG!!!11

File: blog/views.py [hunk 1]
```diff
@@ -1,4 +1,7 @@
 """Views for blog app."""
 from django.shortcuts import render  # noqa: F401
 
-# Create your views here.
+
+def post_list(request):
+    """Render requested page with list of posts."""
+    return render(request, 'blog/post_list.html', {})
```
Comment (webknjaz) [inline blog/views.py]: This PR would be non-atomic without this template in place. Could you please add it as well?

File: blog/urls.py [hunk 2]
```diff
@@ -0,0 +1,7 @@
+"""Django based blog URL configuration."""
```
Comment (webknjaz) [inline blog/urls.py]: Nitpick: I'd call it a URL mapping

File: blog/views.py [hunk 3]
```diff
@@ -1,4 +1,7 @@
 """Views for blog app."""
 from django.shortcuts import render  # noqa: F401
 
-# Create your views here.
+
+def post_list(request):
+    """Render requested page with list of posts."""
```
Comment (webknjaz) [inline blog/views.py]: :+1:

Comment (webknjaz) [reply blog/views.py]: (but, I'd omit this `requested` word, as for me it's too obvious)

**Review by webknjaz — CHANGES_REQUESTED** [review]
High-level purpose of this PR is adding a page with blog posts list. So you need to tell about this in the PR title/description. (description should not repeat title, but add important details, _if needed_)

**Comment by makkaronis4e** [comment]
@webknjaz DONE!

**Review by webknjaz — DISMISSED** [review]

**Review by webknjaz — APPROVED** [review]

**Pull request merged**